### ISSUE TITLE
i18n rel-alternate-hreflang on a website with a base url 

### ISSUE THREAD

**Issue by brumartins** [nuxt-community/sitemap-module#136]
### Version

[2.3.1](https://github.com/sitemap-module/releases/tag/2.3.1)

### Reproduction link

[https://github.com/brumartins/nuxt-wiki](https://github.com/brumartins/nuxt-wiki)

### Steps to reproduce

Hello! :smiley:

I'm trying to use the i18n feature to build a multilingual sitemap, and I'm not getting exactly what I would expect. I'm not sure if I'm missing something in the sitemap-module configuration or if it is a bug.

My Nuxt app is hosted on Github Pages and contains a base url */nuxt-wiki* (http://brumartins.github.io/nuxt-wiki). The routes in the `<loc>` tags are correct, but the alternate links are missing the base url.

Here are the steps to generate the sitemap :
```bash
$ npm install

# generate static project (for deployment on Github Pages)
# select first command if using Windows, otherwise select the 2nd one
$ npm run generate:gh-pages-win 
$ npm run generate:gh-pages
```

Open the `dist/sitemap.xml` file. The sitemap is great, the translated alternate links are taken into account, but the base url is missing for these alternate links.

### What is expected ?

I'd expect a sitemap with

```xml
<url>
	<loc>http://brumartins.github.io/nuxt-wiki/</loc>
	<xhtml:link rel="alternate" hreflang="fr" href="http://brumartins.github.io/nuxt-wiki/fr"/>
	<xhtml:link rel="alternate" hreflang="en" href="http://brumartins.github.io/nuxt-wiki"/>
</url>
```

### What is actually happening?

But I get this (*nuxt-wiki* base url is missing on alternate links)

```xml
<url>
	<loc>http://brumartins.github.io/nuxt-wiki/</loc>
	<xhtml:link rel="alternate" hreflang="fr" href="http://brumartins.github.io/fr"/>
	<xhtml:link rel="alternate" hreflang="en" href="http://brumartins.github.io/"/>
</url>
```

### Additional comments?

Here is the nuxt configuration ([link to full file](https://github.com/brumartins/nuxt-wiki/blob/master/nuxt.config.js))

```js
const HOSTNAME = process.env.DEPLOY_ENV === 'GH_PAGES' ? 'http://brumartins.github.io' : 'http://localhost:3000';

export default {
  /* ... */
  sitemap: {
    hostname: HOSTNAME,
    gzip: true,
    i18n: 'en'
  }
}
```

I tried to add the base url to `hostname` property in the `nuxt.config.js` file. The path in the `<loc>` tag became `http://brumartins.github.io/nuxt-wiki/nuxt-wiki/` and nothing changed for the alternate links.

Thank you in advance for your time :blush:

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/sitemap-module/issues/c103">#c103</a>)</em></sub></div>

**Comment by NicoPennec** [comment]
Hi @brumartins 

Thank you for the feedback, it's an issue on i18n option. I will fix it ASAP.

**Comment by NicoPennec** [comment]
I just published the release [2.3.2](https://github.com/nuxt-community/sitemap-module/blob/dev/CHANGELOG.md#232-2020-06-15) that should solve your issue of `router.base` on i18n hreflangs. Let me know if it's good for you now.

**Comment by brumartins** [comment]
Works like a charm, thanks!
